common/lib/capa/capa/templates/clarification.html

```diff
@@ -0,0 +1,4 @@
+<span class="clarification">
+	<i data-tooltip="${clarification | h}" data-tooltip-show-on-click="true" class="fa fa-info-circle"></i>
```

Comment:
@cptvitamin @sarina Yep, that makes sense. I had `aria-hidden` there originally, but removed it when I changed to the `<i></i>` because I assumed an empty tag would be invisible to a screen reader anyways - but I forgot they will read CSS pseudo element content too.